src/main/java/com/easypost/model/CarrierAccount.java

```diff
@@ -75,6 +76,20 @@ public static CarrierAccount create(Map<String, Object> params, String apiKey) t
 			return request(RequestMethod.POST, classURL(CarrierAccount.class), wrappedParams, CarrierAccount.class, apiKey);
 		}
 
+		// createFedEx
+		public static CarrierAccount createFedEx(Map<String, Object> params) throws EasyPostException {
+			return createFedEx(params, null);
+		}
+		public static CarrierAccount createFedEx(Map<String, Object> params, String apiKey) throws EasyPostException {
+			Map<String, Object> wrappedParams = new HashMap<String, Object>();
+			// the description is required by the endpoint; manually supply it here to ensure that the call goes through
+			params.put("description", "FedEx Account");
+			// wrap the parameters, as required by the endpoint
+			wrappedParams.put("fedex_registration", params);
+			
+			return request(RequestMethod.POST, String.format("%s/%s", EasyPost.API_BASE, "fedex_registrations"), wrappedParams, CarrierAccount.class, apiKey);
```

Comment:
We should be using `carrier_accounts/register` instead of `fedex_registrations`.